### ISSUE TITLE
isochrones/vespa referencing?

### ISSUE THREAD

**Issue by benmontet** [benmontet/k2-characterization#24]
@timothydmorton is there anything that needs to go into the paper for these? Should their presence in the ASCL be cited? Or not since this is the first time they're being presented?

**Comment by timothydmorton** [comment]
Yes, we should put the ASCL references in our paper.

**Comment by benmontet** [comment]
Okay. Can I convince you to add those in?

**Comment by timothydmorton** [comment]
yup, can do.  i've got work to do on the stellar properties section anyway.

**Comment by benmontet** [comment]
Let me know when these are in, I'm just about wrapped up with the referee report. 

What work do you have left for the stellar properties?
